infrastructure/docker/traffic_ops/Dockerfile

```diff
@@ -43,22 +49,27 @@ RUN rm /$(basename $RPM)
 
 RUN cd /opt/traffic_ops/app && carton
```

Comment:
carton requires `POSTGRES_HOME` for `DBD::Pg` to compile correctly since we're using an updated version of Postgres.    `postinstall` does this step already -- can we skip it here?

Reply:
Postinstall is run with `docker run`, not `docker build`. We can omit it here and it will still work, but the goal is to bake as many dependencies into the image as possible, so `docker run` takes as little time to start up as possible.

Reply:
but it still requires the POSTGRES_HOME to be set correctly...

Reply:
Done.